src/transceivercontroller/DefaultTransceiverController.ts

```diff
@@ -586,13 +586,10 @@ export default class DefaultTransceiverController
   }
 
   /**
-   * Adds a transceiver to the peer connection and performs additional necessary setup.
+   * Set up and add a video transceiver to the peer connection.
    */
-  private addTransceiver(
-    trackOrKind: string | MediaStreamTrack,
-    init?: RTCRtpTransceiverInit
-  ): RTCRtpTransceiver {
-    const transceiver = this.peer.addTransceiver(trackOrKind, init);
+  protected setupVideoTransceiver(init?: RTCRtpTransceiverInit): RTCRtpTransceiver {
+    const transceiver = this.peer.addTransceiver('video', init);
```

Comment:
I just did a small refactor with this function to make the intention more clear.